### DISCUSSION TITLE
Exploring Data Transmission Configuration in du_example File

### DISCUSSION THREAD

**Discussion by ahmedelbushra** [srsran/srsRAN_Project#481] [General Help]
I have a question regarding the du_example file. From what I understood, the file transmits only SIB1 message through the PRACH. Can I use the sample example file and configure it so I can transmit a specific amount of data of my choice through the PDSCH to the UE and vice versa?

**Comment by pgawlowicz** [comment]
I do not think it is possible, why you do not want to use gnb?

**Reply by ahmedelbushra** [reply]
Mainly, I want to understand the flow of data within the PDSCH to grasp the working flow of du-low within the code and implement it for my own transmitter and receiver. If I can run only the du-low (phy-high,low,RU) without the need for upper layers, that would be great.

**Reply by pgawlowicz** [reply]
Hi @ahmedelbushra, any progress with your work?
